docs(weekly): add this week in databend 100 (#11946)
datafuselabs/databend: website/blog/2023-07-02-databend-weekly-100.md

---
title: "This Week in Databend #100"
date: 2023-07-02
slug: 2023-07-02-databend-weekly
tags: [weekly]
description: "Stay up to date with the latest weekly developments on Databend!"
contributors:
  - name: ariesdevil
  - name: b41sh
  - name: BohuTANG
  - name: dantengsky
  - name: Dousir9
  - name: drmingdrmer
  - name: everpcpc
  - name: JackTan25
  - name: leiysky
  - name: lichuang
  - name: PsiACE
  - name: RinChanNOWWW
  - name: soyeric128
  - name: sundy-li
  - name: TCeason
  - name: Xuanwo
  - name: xudong963
  - name: youngsofun
  - name: zhang2014
  - name: ZhiHanZ
  - name: zhyass
authors:
  - name: PsiACE
    url: https://github.com/psiace
    image_url: https://github.com/psiace.png
---

[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .

> > Having trouble connecting your root user to Databend? Read [Docs | Configuring Admin Users](https://databend.rs/doc/sql-clients/admin-users) to learn how to set up correctly.

Reviewer: For security reasons, the Root user is no longer available out of the box. You must configure it before use. Learn more at https://databend.rs/doc/sql-clients/admin-users.

## What's On In Databend

Stay connected with the latest news about Databend.

### Announcing Databend v1.2! Data + AI

Databend v1.2 was officially released on June 29, 2023! Thanks to all the community partners who participated and to everyone who contributed to making Databend better!

> The new version has many new features, including:

Reviewer: ```suggestion

```

- New Data Type: `BITMAP`
- Direct Query of CSV/TSV/NDJSON Files Using Column Position
- New Hash Table: Improved Hash Join Performance
- AI Functions
- Computed Columns
- `VACUUM TABLE`
- Serverless Background Service
- Bind `databend` into Python
- BendSQL - Databend Native Command Line Tool
- Integration with Apache DolphinScheduler, Apache Flink CDC and Tableau

If you are interested in learning more, please check out the resources listed below.

- [What's Fresh in Databend v1.2 | Blog | Databend](/blog/databend-changelog-1-2)

## Code Corner

Discover some fascinating code snippets or projects that showcase our work or learning journey.

> ### Introduction to Databend Long Run Tests

Reviewer: ```suggestion
### Databend Long Run Tests
```

> Databend's long run tests aim to verify the correctness and performance of the system under heavy load and concurrency. These tests include concurrent large-scale data ingestion, table maintenance (optimization, re-clustering, and vacuuming), as well as querying.

Reviewer: ```suggestion
Databend's long run tests the correctness and performance of the system under heavy load and concurrency. This includes concurrent large-scale data ingestion, table maintenance (optimization, re-clustering, and vacuuming), as well as querying.
```

> The test will run a series of SQL and validation commands to verify the results. It will begin by executing the pre-test scripts (`_before.sh`), followed by repeatedly running concurrent test scripts, and finally executing post-test scripts (`_after.sh`). All event logs will be stored in a table on Databend for further analysis.

Reviewer: long run test 在这两段话里面的单复数要统一起来，相应的动词要一起变化。
